### PR TITLE
Service/APT: Migrate logging macros

### PR DESCRIPTION
**Pull request by valentinvanelslande**
This change is [Reviewable](https://reviewable.io/reviews/citra-emu/citra/3587)

### CODE REVIEW

**Comment by lioncash** [comment]
Any particular reason why you didn't change line 528 to `{:03X}` or `{:#05X}`?

**Comment by lioncash** [comment]
Can you squash you commits into one? (note that you could have used `git commit --amend` instead of creating a new commit).

**Comment by valentinvanelslande** [comment]
@lioncash how do I put the commit message

File: src/core/hle/service/apt/apt.cpp [hunk 1]
```diff
@@ -460,35 +461,36 @@ void Module::Interface::SetAppCpuTimeLimit(Kernel::HLERequestContext& ctx) {
...
-    LOG_WARNING(Service_APT, "(STUBBED) called value=%u", value);
+    NGLOG_WARNING(Service_APT, "(STUBBED) called, value={}", value);
 }
 
 void Module::Interface::PrepareToStartLibraryApplet(Kernel::HLERequestContext& ctx) {
     IPC::RequestParser rp(ctx, 0x18, 1, 0); // 0x180040
     AppletId applet_id = rp.PopEnum<AppletId>();
 
-    LOG_DEBUG(Service_APT, "called applet_id=%08X", static_cast<u32>(applet_id));
+    NGLOG_DEBUG(Service_APT, "called, applet_id={:#010X}", static_cast<u32>(applet_id));
```
Comment (daniellimws) [inline src/core/hle/service/apt/apt.cpp]: Original one didn't have `0x`, so should be `{:08X}`

File: src/core/hle/service/apt/apt.cpp [hunk 2]
```diff
@@ -505,14 +507,14 @@ void Module::Interface::PrepareToStartNewestHomeMenu(Kernel::HLERequestContext&
...
-    LOG_DEBUG(Service_APT, "called");
+    NGLOG_DEBUG(Service_APT, "called");
 }
 
 void Module::Interface::PreloadLibraryApplet(Kernel::HLERequestContext& ctx) {
     IPC::RequestParser rp(ctx, 0x16, 1, 0); // 0x160040
     AppletId applet_id = rp.PopEnum<AppletId>();
 
-    LOG_DEBUG(Service_APT, "called applet_id=%08X", static_cast<u32>(applet_id));
+    NGLOG_DEBUG(Service_APT, "called, applet_id={:#010X}", static_cast<u32>(applet_id));
```
Comment (daniellimws) [inline src/core/hle/service/apt/apt.cpp]: Original one didn't have 0x, so should be {:08X}

File: src/core/hle/service/apt/apt.cpp [hunk 3]
```diff
@@ -525,7 +527,7 @@ void Module::Interface::FinishPreloadingLibraryApplet(Kernel::HLERequestContext&
     IPC::RequestBuilder rb = rp.MakeBuilder(1, 0);
     rb.Push(apt->applet_manager->FinishPreloadingLibraryApplet(applet_id));
 
-    LOG_WARNING(Service_APT, "(STUBBED) called applet_id=%03X", static_cast<u32>(applet_id));
+    NGLOG_WARNING(Service_APT, "(STUBBED) called, applet_id={:#05X}", static_cast<u32>(applet_id));
```
Comment (daniellimws) [inline src/core/hle/service/apt/apt.cpp]: Shouldn't be `05#`? Because the original one didn't have `0x`

File: src/core/hle/service/apt/apt.cpp [hunk 4]
```diff
@@ -536,7 +538,7 @@ void Module::Interface::StartLibraryApplet(Kernel::HLERequestContext& ctx) {
     Kernel::SharedPtr<Kernel::Object> object = rp.PopGenericObject();
     std::vector<u8> buffer = rp.PopStaticBuffer();
 
-    LOG_DEBUG(Service_APT, "called applet_id=%08X", static_cast<u32>(applet_id));
+    NGLOG_DEBUG(Service_APT, "called applet_id={:#010X}", static_cast<u32>(applet_id));
```
Comment (daniellimws) [inline src/core/hle/service/apt/apt.cpp]: Original one didn't have `0x`, so should be `{:08X}`

File: src/core/hle/service/apt/apt.cpp (hunk 3 above)
Comment (valentinvanelslande) [reply src/core/hle/service/apt/apt.cpp]: applet_id is applet_id in the log?

**Pull request merged**